src/Markdig.Tests/TestReferralLinks.cs

```diff
@@ -37,20 +37,6 @@ public void TestLinksWithCustomRel(string[] rels, string expected)
             Assert.That(html, Contains.Substring($"rel=\"{expected}\""));
         }
 
-
-        [Test]
-        public void TestLinksWithNullRel()
-        {
-            var markdown = "[world](http://example.com)";
-
-            var pipeline = new MarkdownPipelineBuilder()
-                .UseReferralLinks(null)
-                .Build();
-            var html = Markdown.ToHtml(markdown, pipeline);
-
-            Assert.That(html, !Contains.Substring("rel="));
-        }
-
```

Comment:
I don't know if I'm missing something, but I don't understand this test.

If you don't want the `rel` being output... don't add the extension?